src/GraphQLParser/Parser.cs

```diff
@@ -1,22 +1,10 @@
+using System;
 using GraphQLParser.AST;
 
 namespace GraphQLParser
 {
-    public class Parser
+    public static class Parser
     {
-        private readonly ILexer _lexer;
-
-        public Parser(ILexer lexer)
-        {
-            _lexer = lexer;
-        }
-
-        public GraphQLDocument Parse(ISource source) => Parse(source, false);
-
-        public GraphQLDocument Parse(ISource source, bool ignoreComments)
-        {
-            var context = new ParserContext(source, _lexer, ignoreComments);
-            return context.Parse();
-        }
+        public static GraphQLDocument Parse(ReadOnlyMemory<char> source, bool ignoreComments = true) => new ParserContext(source, ignoreComments).Parse();
```

Comment:
@Shane32 I'm sure that in v7 we should ignore comments by default to increase the parsing speed. I will write about it in the readme. Or you if you want.